.github/workflows/release-tagged.yml

```
@@ -0,0 +1,69 @@
+name: Release Tagged
+
+on:
+  repository_dispatch:
+    types: [release-tagged]
+  # TODO: Remove after done testing
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  release-tagged:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Update stackbrew config
+        env:
+          # TODO: Switch to event payload tag
+          TAG: v2.0.0-rc.3
+          # TAG: ${{ github.event.client_payload.tag }}
+        run: |
+          # Get the checksums file from the tagged release
+          wget https://github.com/caddyserver/caddy/releases/download/${TAG}/caddy_${TAG#v}_checksums.txt -O checksums.txt
+
+          # Update the version tag in the config
+          echo "Updating version to '${TAG#v}'"
+          sed -i "s/caddy_version: .*/caddy_version: ${TAG#v}/" stackbrew-config.yaml
```

using `yq` would probably be more reliable:

```suggestion
          yq write -i stackbrew-config.yaml caddy_version ${TAG#v}
```

(I presume `yq` is installed, if not you can run it in a container like the other workflows)